src/Microsoft.DotNet.Build.Tasks.Packaging/src.Desktop/project.json

```diff
@@ -2,9 +2,9 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1",
     "Newtonsoft.Json": "9.0.1",
-    "NuGet.Client": "4.0.0-rc-2129",
```

Comment:
It was merged into `NuGet.Packaging` in https://github.com/NuGet/NuGet.Client/commit/85731166154d0818d79a19a6d2417de6aa851f39:

>  Merging NuGet.ContentModel, NuGet.Client, and NuGet.RuntimeModel into NuGet.Packaging